nala/features/personalization/mep-actions.spec.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
module.exports is an older way of doing things (ES5 JS), you can use ES6 syntax and just export a constant like so
`export const mepActions = { .... ` and then in another file you can import that constant like so: `import { mepActions} from '[path_to_your_file]/mep-actions.spec.js'`